lib/v8.js

```diff
@@ -14,7 +14,10 @@
 
 'use strict';
 
+const Buffer = require('buffer').Buffer;
+
 const v8binding = process.binding('v8');
+const serdesBinding = process.binding('serdes');
```

Comment:
perhaps something more explicit such as `v8serdes` would be better? Not sure about that tho.
@nodejs/node-chakracore have you all considered implementing this mechanism yet?